client-src/index.js

```diff
@@ -243,9 +243,13 @@ var onSocketMessage = {
 		sendMessage("Invalid");
 	},
 	/**
-	 * @param {string} hash
+	 * @param {string | undefined} _hash
```

Comment:
I've modified the parameter name in the type to match the actual parameter being used, so the types are working correctly